app/controllers/fitbit_profiles_controller.rb

```diff
@@ -59,13 +61,17 @@ def show
     
     #grab body measurements for graphs
     if @fitbit_profile.body
-      @body = FitbitBody.find_all_by_fitbit_profile_id(@fitbit_profile.id, :order => "date_logged")
+      @body = FitbitBody
+        .where(fitbit_profile_id: @fitbit_profile.id)
+        .order(:date_logged)
       @bmi = @body.map {|fa| [fa.date_logged, fa.bmi]}
     end
     
     #grab sleep measurements for graphs
     if @fitbit_profile.sleep
-      @sleep = FitbitSleep.find_all_by_fitbit_profile_id(@fitbit_profile.id,:order => "date_logged")
+      @sleep = FitbitSleep
+        .where(fitbit_profile_id: @fitbit_profile.id)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.